### DISCUSSION TITLE
Why store the client into Context rather than using ES6 modules export/import?

### DISCUSSION THREAD

**Discussion by thojanssens** [urql-graphql/urql#2310] [Q&A]
Hello, I wondered why the documentation suggests adding the client into Context, rather then just use ES6 modules and export/import the client. E.g. a `client.js` file:
```javascript
import { createClient } from '@urql/svelte';

export default createClient({ url: 'http://localhost:3000/graphql' });
```

A component that needs the client:
```javascript
import client from './client';

// do something with client
```

**Comment by Av1shay** [comment]
How can you test the components like this? you will have to use the actual client
